src/visualizers/VisualizerFactorHistogram.ts

```diff
@@ -0,0 +1,216 @@
+    // many pixels wide each bin should be
+    binWidth(): number {
+        // 0.95 Creates a small offset from the side of the screen
+        return (0.95 * this.sketch.width) / this.binFactorArray().length
+    }
+
+    // Create a number that represents how many pixels high
+    // each increase of one in the bin array should be
+    height(): number {
+        const binFactorArray = this.binFactorArray()
+        const greatestValue = binFactorArray.reduce(
+            (a, b) => Math.max(a, b),
+            -Infinity
+        )
+        // 0.95 Creates a small offset from the side of the screen
+        return (0.95 * this.sketch.width) / greatestValue
+    }
+
+    draw() {
+        // These numbers provide the rgb values for the background color
+        // This is light blue
+        this.sketch.background(176, 227, 255)
+        this.sketch.textSize(0.02 * this.sketch.height)
+        const height = this.height()
+        const binWidth = this.binWidth()
+        const binFactorArray = this.binFactorArray()
+        const offsetScalar = 0.975
+        const textOffsetScalar = 0.995
+        this.sketch.line(
+            // Draws the y-axis
+            (1 - offsetScalar) * this.sketch.width, // Slightly right of the left edge
+            0,
+            (1 - offsetScalar) * this.sketch.width,
+            this.sketch.height
+        )
+        this.sketch.line(
+            // Draws the x-axis
+            0,
+            offsetScalar * this.sketch.height, // Slightly above the canvas bottom
+            this.sketch.width,
+            offsetScalar * this.sketch.height
+        )
+
+        for (let i = 0; i < binFactorArray.length; i++) {
+            this.sketch.rect(
+                // Draws the rectangles for the Histogram
+                (1 - offsetScalar) * this.sketch.width + binWidth * i,
+                offsetScalar * this.sketch.height
+                    - height * binFactorArray[i],
+                binWidth,
+                height * binFactorArray[i]
+            )
+            if (this.binSize != 1) {
+                // Draws text for if the bin size is not 1
+                this.sketch.text(
+                    this.binSize * i + ' - ' + (this.binSize * (i + 1) - 1),
+                    1 - offsetScalar + binWidth * (i + 1 / 2),
+                    textOffsetScalar * this.sketch.width
+                )
+            } else {
+                // Draws text for if the bin size is 1
+                this.sketch.text(
+                    i,
+                    (1 - offsetScalar) * this.sketch.width
+                        + (binWidth * (i + 1) - binWidth / 2),
+                    textOffsetScalar * this.sketch.width
+                )
+            }
+        }
+
+        this.sketch.noLoop()
+    }
+}
+
+export const exportModule = new VisualizerExportModule(
+    'Factor Histogram',
+    FactorHistogramVisualizer,
+    'Displays a Histogram of the number of prime factors of a sequence.'
+)
```

Comment:
One thing I would love to see (not necessary for this PR) is a curve of the expected distribution for integers graphed overtop of the bars ([like this](https://www.mathworks.com/help/examples/stats/win64/HistogramwithaNormalDistributionFitExample_01.png)).  It might be cool to have error bars on it too, so that you can see whether any bars lying outside an error range of the plotted curve are statistically significant (this would take into account the number of terms).

Reply:
I think we're saving this for later, right?  Marking resolved.